clang/lib/Driver/Action.cpp

```diff
@@ -104,6 +104,12 @@ std::string Action::getOffloadingKindPrefix() const {
     return "device-hip";
   case OFK_SYCL:
     return "device-sycl";
+  case OFK_SYCL_FPGA_IMAGE_AOCO:
+    return "device-fpga-sycl-aoco";
+  case OFK_SYCL_FPGA_IMAGE_AOCR:
+    return "device-fpga-sycl-aocr";
+  case OFK_SYCL_FPGA_IMAGE_AOCX:
+    return "device-fpga-sycl-aocx";
```

Comment:
... This should probably go away then too

Reply:
Reverted